concourse/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -l
+
+set -exo pipefail
+
+function install_plpython3() {
+    if [[ $GP_MAJOR_VERSION == 6 ]]; then
+        mkdir -p bin_plpython3/install_tmp
+        pushd bin_plpython3/install_tmp
+        find .. -maxdepth 1 -regex ".*-[0-9\.]*-.*\.tar\.gz" -exec tar xfv {} \;
+        ./install_gpdb_component
+        popd
+    fi
+}
+
+function _main() {
+    install_plpython3
+
+    source "$CI_REPO_DIR/common/entry_common.sh"
+    sudo passwd --delete gpadmin  # for `su gpadmin` in start_gpdb
+    echo 'alias su="sudo su"' >>"$HOME/.bashrc"
+    source "$HOME/.bashrc"  # for `alias su`
+    start_gpdb
```

Comment:
why not putting `start_gpdb` to `pre_test.sh` like others? https://github.com/pivotal/advanced-password-check/blob/gpdb/concourse/pre_test.sh
- `pre_test.sh` runs with root permission, no need to hack the `su`/`sudo` anymore
- it will add `source` to `.bashrc`, so everything will be set up when gpadmin login.
- Ideally, things in `pre_test.sh` do setup things, and runs only once. So `test.sh` can be run again when hijack to the concourse.

Reply:
Any special reason why not just follow the convention to have a `pre_test.sh`to start gpdb there?

Reply:
OK. saw https://github.com/pivotal/gp-extensions-ci/pull/31#issuecomment-1780352588